.happy/terraform/modules/ecs-stack/county_runs.tf

```diff
@@ -506,11 +442,7 @@ module nextstrain_ventura_contextual_sfn_config {
     remote_dev_prefix        = local.remote_dev_prefix
     group_name               = "Ventura County Public Health Laboratory"
     s3_filestem              = "Ventura Contextual"
-    template_filename        = "group_plus_context.yaml"
-    template_args            = {
-      division = "California"
-      location = "Ventura County"
-    }
+    template_args            = {}
```

Comment:
Thanks for cleaning these up! I've been thinking, we can prob remove some of the counties that never logged in (low priority).